kore/src/Kore/Step/Rule.hs

```diff
@@ -266,18 +269,37 @@ complexRewriteTermToRule attributes pat =
             (TermLike.And_ _
                 (TermLike.Not_ _ antiLeft)
                 (TermLike.And_ _ requires lhs))
-            rhs ->
-                RewriteRule RulePattern
-                    { left = lhs
-                    , antiLeft = Just antiLeft
-                    , requires = Predicate.wrapPredicate requires
-                    , rhs = termToRHS rhs
-                    , attributes
-                    }
+            rhs -> case AntiLeft.parse antiLeft of
+                Nothing -> (error . show. Pretty.vsep)
+                    [ "Cannot parse antileft term"
```

Comment:
```suggestion
                    [ "Cpuld not parse antileft term"
```

"Cannot" sounds to me like it's unsupported, I would have been confused by this message, especially since I don't know what antileft really means.